rasa/shared/core/training_data/story_reader/yaml_story_reader.py

```diff
@@ -336,7 +336,9 @@ def _parse_or_statement(self, step: Dict[Text, Any]) -> None:
                 )
                 return
 
-        self.current_step_builder.add_user_messages(utterances)
+        self.current_step_builder.add_user_messages(
+            utterances, self.is_used_for_conversion
```

Comment:
`is_used_for_conversion` doens't make really sense with our usage. We only generated checkpoints / new stories for training, correct? How about naming it something like `is_used_for_training`?

Reply:
Or `is_used_for_displaying`?

Reply:
I can change the name, but that will change the reader's public interface since the attribute does not start with `_`. Do you think we should change it anyways?

Reply:
How about `expand_or_statements`? A bit more precise, since it describes what is being done, rather the use that the user will give it.

Reply:
> I can change the name, but that will change the reader's public interface since the attribute does not start with _. Do you think we should change it anyways?

I don't think that anything besides our code is already using that. Imo it makes also sense to make it private / protected. 

> How about expand_or_statements? A bit more precise, since it describes what is being done, rather the use that the user will give it.

The problem is that it does not just used for that. In [this](https://github.com/RasaHQ/rasa/blob/8980e0470644e4ece67c82cc195bf3a73c88f123/rasa/shared/core/training_data/story_reader/markdown_story_reader.py#L234) case it's also used for something else 😬   `is_used_for_training` seems more general and [intent revealing](https://wiki.c2.com/?IntentionRevealingNames) in that case.